app/controllers/katello/api/v2/package_groups_controller.rb

```diff
@@ -3,6 +3,46 @@ class Api::V2::PackageGroupsController < Api::V2::ApiController
     apipie_concern_subst(:a_resource => N_("a package group"), :resource => "package_groups")
     include Katello::Concerns::Api::V2::RepositoryContentController
 
+    before_action :find_repository, :only => [:create, :destroy]
```

Comment:
this actually breaks package group listing for a single repository, as this same before_action is defined here: https://github.com/Katello/katello/blob/master/app/controllers/katello/concerns/api/v2/repository_content_controller.rb#L9

and this is overriding it.  you can just leave this out, or just use `before_action :find_repository`